pkg/controller/config/v1alpha1/defaults.go

```diff
@@ -15,5 +17,6 @@ func SetDefaults_ExtensionConfig(obj *ExtensionConfig) {
 func SetDefaults_NTPConfig(obj *NTPConfig) {
 	if obj.Daemon == "" {
 		obj.Daemon = SystemdTimesyncd
+		obj.Enabled = true
```

Comment:
That dose not work. If I set `enabled` to `false` without setting a `daemon`, this will set `enabled` to `true` again.